src/components/menu/ComingSoon/GoogleBrandFormatting.css

```
@@ -0,0 +1,175 @@
+/* 
+https://brandpalettes.com/google-colors/ I decided to use brand colours of google to keep theme consistent - A.N.
+CSS Style inspired by Bold by Rafal Tomal https://wdexplorer.com/20-examples-beautiful-css-typography-design/
+*/
+
+:root {
+  --google-blue: #4285f4;
+  --google-red: #db4437;
+  --google-yellow: #f4b400;
+  --google-green: #0f9d58;
+  --google-white: #ffff;
+}
+
+.center {
+  margin: auto;
+  width: 60%;
+  border: 5px solid var(--google-blue);
+  padding: 10px;
+}
+
+/* Header 1 from Google Blue to Google White  */
+.h1-blue {
+  color: var(--google-blue);
+  font-family: "Helvetica Neue", sans-serif;
+  font-size: 65px;
+  font-weight: bold;
+  letter-spacing: -1px;
+  line-height: 1;
+  text-align: center;
+}
+
+.h1-red {
+  color: var(--google-red);
+  font-family: "Helvetica Neue", sans-serif;
+  font-size: 65px;
+  font-weight: bold;
+  letter-spacing: -1px;
+  line-height: 1;
+  text-align: center;
+}
```

There's a lot of repetition between these classes. How about we create a common class so that you can combine it with color variations. I also have questions to naming: why did you choose `.h1-color`, if the elements you apply these classes to are `span`s? I would go with just a generic `.text-google-color`.
```suggestion
.google-text {
  font-family: "Helvetica Neue", sans-serif;
  font-size: 65px;
  font-weight: bold;
  letter-spacing: -1px;
  line-height: 1;
  text-align: center;
}

.text-google-blue {
  color: var(--google-blue);
}

.text-google-red {
  color: var(--google-red);
}
```

---

I removed repetitive code. I made generic classes for colour, as well as text-align!